roles/pulibrary.nginxplus/ADDHOSTS.md

```diff
@@ -0,0 +1,39 @@
+Adding new hosts to Nginx Plus
+==============================
+
+* You will need to register the new Virtual Server at
+  [https://networkregistration.princeton.edu](https://networkregistration.princeton.edu) - for now this is a bug because permission/authority is required from OIT, so
+ping the #Operations Slack channel. The turn around is instant + time for DNS to
+propagate worldwide.
+
+* Generate SSL keys for the Virtual Server above. This is also another bug that
```

Comment:
Is this a manual process and the slack ping is the only step.  In the future there will be a non manual step?

Reply:
this is a desired goal but it is not under our control.